runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -3509,6 +3513,54 @@ MM_CopyForwardScheme::scanUnfinalizedObjects(MM_EnvironmentVLHGC *env)
 }
 #endif /* J9VM_GC_FINALIZATION */
 
+void
+MM_CopyForwardScheme::scanContinuationObjects(MM_EnvironmentVLHGC *env)
+{
+	/* ensure that all clearable processing is complete up to this point since this phase resurrects objects */
```

Comment:
This comment is not correct (this phase does not resurrect objects, unlike Unfinalized which does). 
And sync is not needed (there was a sync at the complete phase of whichever last clearable step really did resurrect objects).